### PR TITLE
fix: add api for datastore/admin

### PR DESCRIPTION
**Pull request by dmitry-fa**
This request is only for information, do not merge.
It contains changes to synth.py and generated files.

### CODE REVIEW

**Comment by BenWhitehead** [comment]
We can't merge this until other work toward v2 has landed. I've tagged it as blocked and targeting milestone v2.

**Comment by meredithslota** [comment]
@BenWhitehead v2 has been out for some time — is this PR still relevant?

**Review by suztomo — DISMISSED** [review]

**Pull request closed without merging**
